client/src/components/SearchableDropdownField/SearchableDropdownField.js

```diff
@@ -166,12 +166,12 @@ SearchableDropdownField.propTypes = {
   multi: PropTypes.bool.isRequired,
   name: PropTypes.string.isRequired,
   placeholder: PropTypes.string.isRequired,
-  onChange: PropTypes.func.isRequired,
+  onChange: PropTypes.func,
```

Comment:
I don't think that this method is always required. [This recommendation](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/require-default-props.md) gives advice to use `defaultProps` instead of `isRequired` if we are not sure that this value will be provided.